src/client/datascience/export/fileConverter.ts

```diff
@@ -218,6 +218,11 @@ export class FileConverter implements IFileConverter {
                 'Unable to export notebook as the built-in vscode.ipynb extension is currently unavailable.'
             );
         }
+        // Via the interactive window export this might not be activated
+        if (!serializerApi.isActive) {
```

Comment:
In my export refactor I made Interactive Window use this method to get the contents from the document (it's just in the nbconvert path) however if you've only used IW then the core notebook extension might be available, but inactive and it would crash at the later serializerApi.exports call. Wasn't an issue for notebooks since if you had a notebook open then vscode.ipynb would be guaranteed to be active already.